File: ejemplo_api/src/main/java/ni/edu/ejemplo_api/models/Persona.java
package ni.edu.ejemplo_api.models;

/**
 * Project: ejemplo_api
 * From: ni.edu.ejemplo_api.models
 * Created by: Sara Zambrana
 * On: 04/05/2026 at 18:06
 * All rights reserved 2026.
 */
public class Persona {
    private String nombre;
    private int edad;

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    public Persona(String nombre, int edad) {
        this.nombre = nombre;
        this.edad = edad;
    }

    public void put(String mensaje, String esMenorDeEdad) {

    }
}

File: Ejercicios_Sesion#03/Ejercicio09-HerenciaSistemaEducativo/src/Persona.kt
/**
 * Project: Ejercicio09-HerenciaSistemaEducativo
 * From:
 * Created by: Sara Zambrana
 * On: 17/03/2026 at 13:30
 * All rights reserved 2026.*/

open class Persona (val nombre:String,
                    var edad : Int){
    fun infoPersona() : String{
        return "Nombre: $nombre, Edad: $edad"
    }
}